docs/users.rst

```diff
@@ -170,9 +170,38 @@ be provided, or "pulp_repos" set to true. Otherwise, build will fail.**
 provided, "modules" will be ignored. "pulp_repos" can be used by itself, or with
 either "packages" or "modules".**
 
+.. _container.yaml-autorebuild:
+
+autorebuild
+~~~~~~~~~~~
+
+This section specifies whether and how a build should be rebuilt based on
+changes to the base parent image.
+
+TODO
```

Comment:
Is this a place-holder for filling out more about container.yaml keys?

Reply:
Well, more about autorebuilds specifically (got any seed info?), but it would be nice to describe all available members.

Reply:
Unfortunately the [schema|https://github.com/projectatomic/atomic-reactor/blob/1.6.33/atomic_reactor/schemas/container.json#L21] is silent on the matter, but the [code|https://github.com/projectatomic/atomic-reactor/blob/1.6.33/atomic_reactor/plugins/pre_check_and_set_rebuild.py#L112] says there is a single key, "from_latest". @lcarva, is that described somewhere?

Reply:
No, I don't think it's externally described anywhere.

Reply:
Mentioned here: https://github.com/projectatomic/atomic-reactor/pull/1056